package.json

```diff
@@ -46,6 +46,7 @@
     "@babel/node": "^7.2.2",
     "@babel/plugin-proposal-class-properties": "^7.2.1",
     "@babel/plugin-proposal-object-rest-spread": "^7.2.0",
+    "@babel/plugin-syntax-class-properties": "^7.2.0",
```

Comment:
Babel was unable to find this plugin when I didn't install it directly

Reply:
I think this is due to the duplicate `@babel/core`. I might have a solution for this.

Reply:
Deduped `@babel/core` which means we're using it internally as well now.